apollo-backend/src/main/java/io/logz/apollo/notifications/ApolloNotifications.java

```diff
@@ -71,6 +73,18 @@ public void notify(Deployment.DeploymentStatus status, Deployment deployment) {
                 });
     }
 
+    private boolean filterNotificationByType(Notification notification, Deployment.DeploymentStatus status) {
+        if (notification.getType() == Notification.NotificationType.SLACK && status == Deployment.DeploymentStatus.STARTED) {
```

Comment:
can you explain this condition please?
why do you filter out SLACK notifications?

Reply:
notifications doesn't currently support any other filter than scopes like environment/service ->
I wanted to add a simple mechanism to be able to filter out notifications that doesn't belong to the scope that I want, i.e, I don't "STARTED" notifications sent to Slack, and I don't want "DONE/CANCELED" notifications sent to the markers-service (currently).

I think that we can create a better solution, like each class will have a method that act as a template method for this filtering, I just wanted a small implementation for now. wdyt?

Reply:
I see, I think it's ok for now but each notification should provide a list of excluded deployment statuses
it could be a field in `notification_json_configuration`
@mayhuberman wdyt?

Reply:
Maybe it's better to hold it in `notification_json_configuration` because if we'll have more notifications types it will complicate stuff. But for now it's fine by me :)

Reply:
imo it should be stored in a different field, like "NOTIFY_ON_STATUS" and a list of statuses (or blank, which will send to all) that should be used, but whatever you think's best